JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

```diff
@@ -393,7 +393,7 @@ def is_finished(reservation):
             count += len(reservation.data_reservation.containers)
             count += len(reservation.data_reservation.kubernetes)
             count += len(reservation.data_reservation.proxies)
-            count += len(reservation.data_reservation.reserve_proxies)
+            count += len(reservation.data_reservation.reverse_proxies)
```

Comment:
there's another one in this file

Reply:
@grimpy  Line 855 does not exist?

Reply:
https://github.com/threefoldtech/jumpscaleX_libs/blob/development/JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py#L885

Reply:
Wait, what? I can't see that... This code is not checked in on my branch: 
https://github.com/threefoldtech/jumpscaleX_libs/blob/development_fix_reverseproxy_spelling/JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

Reply:
I assume you branched of from an old revision than, rebase and you will have it...